Formula/sbcl.rb

```diff
@@ -6,6 +6,12 @@ class Sbcl < Formula
   license all_of: [:public_domain, "MIT", "Xerox", "BSD-3-Clause"]
   head "https://git.code.sf.net/p/sbcl/sbcl.git", branch: "master"
 
+  livecheck do
+    url "https://sourceforge.net/projects/sbcl/files/sbcl/"
+    regex(%r{href=.*?/files/sbcl/(\d+(?:[.-]\d+)+)/}i)
+    strategy :page_match
```

Comment:
```suggestion
    url "https://sourceforge.net/projects/sbcl/rss?path=/sbcl"
```

In this case, it's not necessary to check the files page. We simply need to check the RSS feed for the `/sbcl` subdirectory instead of the main RSS feed. This can be done by appending the `path` query string parameter to the RSS feed URL.

Reply:
👍